src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSIApplication.java

```diff
@@ -638,4 +648,141 @@ public void verifyThatTheGuardarButtonIsNotAvailableIfTheNombreDeAlmacenFieldIsE
     public void hePageLoadedMustContainAsTitle(String title) throws Throwable {
         Assert.assertEquals(storageReport.getTitle(), title, String.format(ErrorMessage.ERROR_MESSAGE_NAME_ELEMENT_PRESENT, title));
     }
+
+    @And("^Click 'Login' button on Page Header$")
+    public void clickLoginButtonOnPageHeader() throws Throwable {
+        if (navigationBar == null) {
+            login.clickLoginButton();
+        }
+    }
+
+    @And("^Enter Username$")
+    public void enterUsername() throws Throwable {
+        if (navigationBar == null) {
+            login.enterUsername();
+        }
+    }
+
+    @And("^Enter Password$")
+    public void enterPassword() throws Throwable {
+        if (navigationBar == null) {
+            login.enterPassword();
+        }
+    }
+
+    @And("^Click 'Login' button to accept credentials$")
+    public void clickLoginButtonToAcceptCredentials() throws Throwable {
+        if (navigationBar == null) {
+            navigationBar = login.clickLoginAcceptButton();
+        }
+    }
+
+    @And("^Click 'Activos' menu in Navigation bar$")
+    public void clickActivosMenuInNavigationBar() throws Throwable {
+        activosMenu = navigationBar.clickActivos();
+    }
+
+    @And("^Click 'Lista de Tipos' submenu option$")
+    public void clickListaDeTiposMenuOption() throws Throwable {
+        listItemType = activosMenu.clickItemTypeListSubMenu();
+    }
+
+    @And("^Click 'Edit' button of the row \"([^\"]*)\" in the list$")
+    public void clickEditButtonOfTheRowInTheList(String indexParam) throws Throwable {
+        int index = Integer.parseInt(indexParam) - 1;
+        editItemType = listItemType.clickEditBtnInRow(index);
+    }
+
+    @And("^Set \"([^\"]*)\" as the new name for the item type$")
+    public void setAsTheNewNameForTheItemType(String newName) throws Throwable {
+        editItemType.enterNewItemTypeName(newName);
+    }
+
+    @When("^Click 'Save' button$")
+    public void clickSaveButton() throws Throwable {
+        listItemType = editItemType.clickSaveItemTypeBtn();
+    }
+
+    @Then("^Verify row \"([^\"]*)\" has \"([^\"]*)\" as name in Item Type list page$")
+    public void verifyRowHasAsNameInItemTypeListPage(String indexParam, String expectedItemTypeName) throws Throwable {
+        int index = Integer.parseInt(indexParam) - 1;
+        String actualItemTypeName = listItemType.getItemTypeNameInTableRow(index);
+        Assert.assertEquals(actualItemTypeName, expectedItemTypeName);
```

Comment:
Let's use the error messages for each assert. For example:
 @Then("^verify that new Contingency plan is added in the list$")
    public void verifyContingencyPLantWasAdded() throws Throwable {
        Assert.assertEquals(contingencyPlan.countContingencies(), totalContingencyPlans + NUMBER_ROW, String.format(ErrorMessage.ERROR_MESSAGE_CREATE_CONTINGENCY_PLAN, "contigency"));